src/lib.rs

```diff
@@ -86,15 +88,16 @@ where
 /// let mut source = File::open("tree.tar.gz")?;
 /// let dest = Path::new("/tmp/dest");
 ///
-/// uncompress_archive(&mut source, &dest)?;
+/// uncompress_archive(&mut source, &dest, true)?;
```

Comment:
I'd prefer a more meaningful argument as an enum.

Something like:
```rust
enum Options {
    PerserveOwnership,
    LossyOwnership,
}
```

Reply:
Since the API has only one option, this seems to me over exaggerated, this will be more helpful when more options can be configured. But I can change for now.

Reply:
It might look as overkill however the `true` does not communicate to the API user what it is.